api/src/main/java/com/codeforcommunity/dto/site/SiteEntry.java

```diff
@@ -0,0 +1,300 @@
+  private final Boolean bicycle;
+  private final Boolean bagEmpty;
+  private final Boolean bagFilled;
+  private final Boolean tape;
+  private final Boolean suckerGrowth;
+  private final String siteType;
+  private final String sidewalkWidth;
+  private final Double siteWidth;
+  private final Double siteLength;
+  private final String material;
+  private final Boolean raisedBed;
+  private final Boolean fence;
+  private final Boolean trash;
+  private final Boolean wires;
+  private final Boolean grate;
+  private final Boolean stump;
+  private final String treeNotes;
+  private final String siteNotes;
+
+  public SiteEntry(
+      Integer id,
+      String username,
+      Timestamp updatedAt,
+      Boolean treePresent,
+      String status,
+      String genus,
+      String species,
+      String commonName,
+      String confidence,
+      Double diameter,
+      Double circumference,
+      Boolean multistem,
+      String coverage,
+      String pruning,
+      String condition,
+      Boolean discoloring,
+      Boolean leaning,
+      Boolean constrictingGate,
+      Boolean wounds,
+      Boolean pooling,
+      Boolean stakesWithWires,
+      Boolean stakesWithoutWires,
+      Boolean light,
+      Boolean bicycle,
+      Boolean bagEmpty,
+      Boolean bagFilled,
+      Boolean tape,
+      Boolean suckerGrowth,
+      String siteType,
+      String sidewalkWidth,
+      Double siteWidth,
+      Double siteLength,
+      String material,
+      Boolean raisedBed,
+      Boolean fence,
+      Boolean trash,
+      Boolean wires,
+      Boolean grate,
+      Boolean stump,
+      String treeNotes,
+      String siteNotes) {
+    this.id = id;
+    this.username = username;
+    this.updatedAt = updatedAt;
+    this.treePresent = treePresent;
+    this.status = status;
+    this.genus = genus;
+    this.species = species;
+    this.commonName = commonName;
+    this.confidence = confidence;
+    this.diameter = diameter;
+    this.circumference = circumference;
+    this.multistem = multistem;
+    this.coverage = coverage;
+    this.pruning = pruning;
+    this.condition = condition;
+    this.discoloring = discoloring;
+    this.leaning = leaning;
+    this.constrictingGate = constrictingGate;
+    this.wounds = wounds;
+    this.pooling = pooling;
+    this.stakesWithWires = stakesWithWires;
+    this.stakesWithoutWires = stakesWithoutWires;
+    this.light = light;
+    this.bicycle = bicycle;
+    this.bagEmpty = bagEmpty;
+    this.bagFilled = bagFilled;
+    this.tape = tape;
+    this.suckerGrowth = suckerGrowth;
+    this.siteType = siteType;
+    this.sidewalkWidth = sidewalkWidth;
+    this.siteWidth = siteWidth;
+    this.siteLength = siteLength;
+    this.material = material;
+    this.raisedBed = raisedBed;
+    this.fence = fence;
+    this.trash = trash;
+    this.wires = wires;
+    this.grate = grate;
+    this.stump = stump;
+    this.treeNotes = treeNotes;
+    this.siteNotes = siteNotes;
+  }
+
```

Comment:
🏕️ I know theres a lot of stuff here, but some javadoc might be useful in the future to know what these fields represent. Not super important since most of these are self explanatory

Reply:
All the non-self explanatory ones are documented in the documentation